### DISCUSSION TITLE
Can the `$` model select behaviour be disabled?

### DISCUSSION THREAD

**Discussion by oalders** [CopilotC-Nvim/CopilotChat.nvim#585] [Q&A] [answered]
This is making it really hard for me to enter variable names which require a `$`.

https://asciinema.org/a/iYKvCslMyhur52tStG6ceXlii

**Comment by deathbeam** [comment, marked as answer]
With that completion configuration you will probably have issues with the other symbols as well ocassionally so imo its better to either disable autocomplete (there is config option for that, `chat_autocomplete`) or configure completeopt so it do not auto selects. here is mine for example:

```
vim.o.completeopt = 'menu,menuone,noinsert,fuzzy,popup'
```

(idr if fuzzy and popup are in neovim release so can just skip those 2)

**Reply by oalders** [reply]
Thank you. It looks like `chat_autocomplete` is what I was looking for.

**Reply by deathbeam** [reply]
btw i also just merged this so the default behaviour will at least be a lot better now and wont complete and insert automatically

https://github.com/CopilotC-Nvim/CopilotChat.nvim/commit/3766c00fa9850b385c6a5077d286ec75f2aeabf1
